packages/useValidators/README.md

```diff
@@ -0,0 +1,44 @@
+# useValidators
+
+This is a hook that allows the definition of validators that can be used to run against a provided value. 
+
+# Installation
+
+### npm
+```bash
+npm install @es-hooks/useValidators
+```
+
+### yarn
+```bash
+yarn add @es-hooks/useValidators
+```
+
+# Usage
+
+```jsx
+import React, { useContext } from 'react'
+import useValidators from '@es-hooks/useValidators'
+
+function MyComponent(props) {
+  function isOverThreeCharacters(value) {
+    return value.length > 3 ? true : false
+  }
+
+  const [ValidationContext, invokeValidators] = useValidators([isOverThreeCharacters])
+  const results = useContext(ValidationContext)
```

Comment:
Is there a reason to use `createContext` around the validation results? I'm struggling to think of a way to pass `ValidationContext` around so child components can consume it directly. In this example `useContext` seems like an unnecessary step if we can just return the validationResults directly instead of `ValidationContext`.

Reply:
I think that's fair. I originally thought it might be useful, but I think returning the results will work fine and if people need to put it into some kind of context object, they can do that themselves.